### PR TITLE
Add cull for tx pool

### PR DESCRIPTION
**Pull request by u2**

### CODE REVIEW

File: tx_pool/src/pool.rs [hunk 1]
```diff
@@ -239,6 +238,16 @@ impl Pool {
         tx_list
     }
```
Comment (keroro520) [inline tx_pool/src/pool.rs:240]: Not sure that could we add `#cfg(test)` here ?

**Review by chaoticlonghair — DISMISSED** [review]

File: env.sh [hunk 2]
```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
```
Comment (chaoticlonghair) [inline env.sh]: I still think this script is not necessary. 
(And, without README, no one could understand what it is.)

File: tx_pool/src/pool.rs (hunk 1 above)
Comment (u2) [reply tx_pool/src/pool.rs:240]: No, this is a pub function.

File: env.sh (hunk 2 above)
Comment (u2) [reply env.sh]: Make sense.

**Review by chaoticlonghair — APPROVED** [review]

**Pull request merged**